cmd/kops/validate_cluster.go

```diff
@@ -222,7 +222,7 @@ func RunValidateCluster(ctx context.Context, f *util.Factory, out io.Writer, opt
 					time.Sleep(pollInterval)
 					continue
 				} else {
-					return nil, fmt.Errorf("cluster passed validation %d consecutive times", consecutive)
+					return nil, fmt.Errorf("cluster passed validation %d consecutive times, failed because of negative options.wait %d", consecutive, options.wait)
```

Comment:
This message is inaccurate. Options.wait is not necessarily negative. 

The lack of a wait option is not a cause of the cluster's failure to pass validation.

Reply:
Hi @johngmyers , you mean when it fails, Options.wait is not necessarily negative.?  I just noticed ` if options.wait > 0 {` , so what's your recommended error message for it? thanks

Reply:
> so what's your recommended error message for it? thanks

I is recommended to leave it as it is, without changes.